docs/developer_guide.md

```diff
@@ -46,25 +49,67 @@ requirements:
   gcloud auth application-default login
   ```
 
-### Develop with Kubernetes
+### Package javascript and static assets
+
+```bash
+./run_npm.sh
+```
 
-This is only needed for development with a local Kubernetes cluster (e.g. for
-changes dependent on mixer changes).
+This will watch static files change and re-build on code edit.
 
-- Install the following tools:
+### Start the Flask Server
 
-  - [`Docker`](https://www.docker.com/products/docker-desktop)
-  - [`Minikube`](https://minikube.sigs.k8s.io/docs/start/)
-  - [`Skaffold`](https://skaffold.dev/docs/install/)
-  - [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-  - [`kustomize`](https://kustomize.io/)
-  - [`yq` 4.x](https://github.com/mikefarah/yq#install)
+Start the flask webserver locally at localhost:8080
+
+```bash
+./run_server.sh
+```
+
+If you don't have access to DataCommons maps API, run
+
+```bash
+./run_server.sh -e lite
+```
+
+This brings up website without place search functionality.
+
+There are multiple environments for the server, specified by `-e` options.
+For example, `custom` is for custom data commons and `iitm` is
+for iitm data commons.
+
+To start multiple instances, bind each server instance to a different port.
+The following example will start localhost on port 8081. The default is 8080.
+
+```bash
+./run_server.sh -p 8081
+```
+
+Please note the strict syntax requirements for the script, and leave a space
+after the flag. So: `./run_server.sh -p 8081` but not `./run_server.sh -p=8081`.
+
+## Deploy local instance to GCP (dev insance)
+
+Commit all changes locally, so the local change is identified by a git hash.
+Then run
+
+```bash
+gcloud auth login
+gcloud auth configure-docker
+./scripts/push_image.sh
+./scripts/deploy_gke.sh dev us-central1
+```
+
+The script builds docker image locally and tag it with the local git commit
```

Comment:
nit: ...and tags it with ..., then deploys to dev ...

Reply:
Done